### ISSUE TITLE
How to save the result of text processing 

### ISSUE THREAD

**Issue by gabo-di** [catboost/catboost#2128]
Hello,

I am using catboost in python for a research. I have a text dataset, and I have to make many classification tasks for the same text but with different labels. Is there a way that I can save the result of text processing so I do not have to make it each time. 

For example I start with

```
labels = make_labels( params )
text_processing = make_processing_dict( )
metrics = [ ]

df_train ,df_test, y0_train, y0_test = train_test_split(df, labels, test_size=0.2, random_state=seed)
learn_pool = Pool(df_train, y0_train, text_features=['clean_text'], feature_names=list(df_train))
test_pool = Pool(df_test, y0_test, text_features=['clean_text'], feature_names=list(df_train))


model_cat = CatBoostClassifier(loss_function='MultiClass', random_seed=seed, eval_metric='Accuracy', text_processing=text_processing)
model_cat.fit(learn_pool,eval_set=test_pool,verbose=100)
metrics = save_metric(metrics, model, df_test, y0_test)
```
(Now I want to change the labels and save the new metrics)

Since the text is not changing and I need to do repear this task many times, I would like to know if there is a way to save the preprocessed text, and maybe export it to a data frame so that would be my new input?  (I know I can preprocess the text in other ways, but if I can just save what CatBoost already does it would be nice )
